server/sonar-web/src/main/js/apps/projectActivity/utils.js

```diff
@@ -141,10 +141,18 @@ export const getAnalysesByVersionByDay = (
   byDay: { [string]: Array<Analysis> }
 }> =>
   analyses.reduce((acc, analysis) => {
-    if (acc.length === 0) {
-      acc.push({ version: undefined, key: undefined, byDay: {} });
+    let currentVersion = acc[acc.length - 1];
+    const versionEvent = analysis.events.find(event => event.category === 'VERSION');
+    if (versionEvent && versionEvent.category === 'VERSION') {
```

Comment:
True, I don't need to, it was just a dumb copy past of the previous condition, I'm gonna remove it, thanks!